packages/playground/src/views/not_yet_implemented.vue

```diff
@@ -1,9 +1,43 @@
 <template>
-  <div>not_yet_implemented</div>
+  <img class="centered-img" src="images/404.png" />
+  <p class="text1">Oops. The page you were looking for doesn't exist.</p>
+  <p class="text2">You may have mistyped address or the page may have been moved</p>
+  <div style="text-align: center">
+    <a href="/" class="link-text"> Take me back to the home page</a>
+  </div>
```

Comment:
Please use `v-btn` with *color="primary"* and *variant="text"* to get the require style
also make sure to inject $router using setup function to use vue router `$router.push("/")`